examples/get.js

```diff
@@ -8,7 +8,7 @@ const solr = require('./../lib/solr');
 const client = solr.createClient();
 
 const query = 'q=id:UTF8TEST&mlt.fl=manu,cat&mlt.mindf=1&mlt.mintf=1';
-client.get('mlt', query, function (err, obj) {
+client.doQuery('mlt', query, function (err, obj) {
```

Comment:
why replace `get` with `doQuery`?

Reply:
I'm not a fan of this change, TBH. In my experience, trying to have one super-generic method for doing calls with any HTTP verb ends up being very messy. It will become even worse after we go to TypeScript, as call signatures for GET, POST/PATCH/PUT and DELETE are very different.

Reply:
I did not replace it. I just renamed the function, and then refactored the contents. It's still doing the exact same thing as before.

It's also not "super generic". The original solr-client code has this logic, which I did not want to change, because I'm refactoring the code, not rewriting it:

- Build a get query
- Just before sending it, check if the URL would be too long
- If so, throw it away and build a POST query with the query string in the body (not JSON)
- Send the query

This logic is very specific to solr queries, so the word "query" in "doQuery" refers to a Solr query object, not a generic HTTP query.